recipes/mosdepth/meta.yaml

```diff
@@ -1,27 +1,25 @@
 {% set version = "0.3.3" %}
-{% set sha256hash = "59d302d60037763d632ce7e5daa1300058d92a4eea92637b6f40047d0b1c56fd" %}
 
 package:
   name: mosdepth
   version: {{ version }}
 
-source:
-  - url: https://github.com/brentp/mosdepth/releases/download/v{{ version }}/mosdepth
-    sha256: {{ sha256hash }}
-
 build:
-  number: 1
-  noarch: generic
+  number: 2
 
 requirements:
   build:
+    - {{ compiler('c') }}
+    - curl
+    - nim
   host:
+    - htslib >=1.10
+    - pcre
```

Comment:
is pcre required?

Reply:
Good catch! I don't think it is. I will remove it in the next commit.